tests/dummy/app/templates/configurable-columns.hbs

```diff
@@ -82,14 +81,22 @@
     </div>
 
     <div class="col-md-12 bumper-30">
-      <h3>application_controller.coffee</h3>
+      <h3>Controller</h3>
       <div class="highlight">
-<pre>App.ApplicationController = Ember.Controller.extend
+<pre class="lang-coffee">App.ApplicationController = Ember.Controller.extend
```

Comment:
We probably want to update this documentation to use the ES6 module syntax. This is not relevant for an Ember-CLI application.